osmtm/templates/task.state.done.mako

```diff
@@ -18,16 +18,32 @@ done = task.cur_state and task.cur_state.state == TaskState.state_done
       <i class="glyphicon glyphicon-thumbs-down icon-white"></i> ${_('Invalidate')}
     </button>
     % if done:
+      % if user == task.states[0].user and not (user.is_admin or user.is_project_manager):
+<%
+validation_message = _("You cannot validate a task that you have marked done.")
+%>
+      <button type="submit" rel="tooltip"
+              name="validate"
```

Comment:
I guess some of these tags aren't absolutely necessary on the button, but I kept them in to be consistent with the enabled one (sans the tooltip, which will not show up if you hover over a disabled button).